src/foundry/client/pixi/layers/controls/layer.d.ts

```diff
@@ -13,6 +13,11 @@ declare global {
   class ControlsLayer extends InteractionLayer {
     constructor();
 
+    /**
+     * @remarks This is not overridden in foundry but reflects the real behavior.
```

Comment:
I think this should be a privateRemarks unless people outside of types maintenance will care.

Reply:
I'll adjust all of these to match that